### DISCUSSION TITLE
Clicking on a field reveals it in the Preview Window

### DISCUSSION THREAD

**Discussion by edhans** [TabularEditor/TabularEditor3#1039] [Ideas]
**Is your feature request related to a problem? Please describe.**
When I right-click on a table and select Preview Data, if I then single click on a field the Preview Window should scroll to and highlight that field in some way just like Power BI Desktop does in the Data view. This can be especially useful for really wide DIM tables.

**Describe the solution you'd like**
A single-click on a field - note that a double-click on a calc column would still bring up the DAX Expression editor.

**Describe alternatives you've considered**
None

**Comment by DBojsen** [comment]
Hi @edhans 

Makes a lot of sense - I have added it to the backlog.

Best Regards

David - Tabular Editor 3 Support

**Comment by corycundy** [comment]
I really like this idea. 

To build off of this, it would also be helpful for the columns in the Preview Window to show up in the order they appear in the TOM explorer (which seems to be alphabetical).

It appears that they currently show up based on the ID in the $SYSTEM.TMSCHEMA_COLUMNS DMV which is seemingly random unless specifying columns manually in the partition. I wish Tabular could bring those in based on the order of the columns in the underlying data source especially if it is a table or view.

**Comment by mlonsk** [comment]
This feature has been released in Tabular Editor 3.27.0.
